Expensify/App
# refactor checkbox with label

### Details


### Fixed Issues

$ https://github.com/Expensify/App/issues/7531

### Tests

1. Run npm run storybook
2. Try submitting the form
3. Check if shows error

### QA Steps




### Tested On


### Screenshots


#### Web


https://user-images.githubusercontent.com/83179295/153957705-075c09b2-46c5-447a-857d-642fa7c83719.mov

without margin left-
<img width="919" alt="Screenshot 2022-02-15 at 2 38 34 AM" src="https://user-images.githubusercontent.com/83179295/153958137-570b6718-fc87-44d9-870d-e4e1ab37ca20.png">

with margin left-

https://user-images.githubusercontent.com/83179295/153959144-2271e08b-11a3-454b-a8bb-50814c3f33e7.mov

#### Mobile Web
<img width="487" alt="Screenshot 2022-02-15 at 2 47 34 AM" src="https://user-images.githubusercontent.com/83179295/153957765-850ac7db-b5c4-483f-a5b0-2d9464fcbad6.png">





#### Desktop

<img width="821" alt="Screenshot 2022-03-01 at 10 09 20 PM" src="https://user-images.githubusercontent.com/83179295/156210466-b7b2fefb-3ad8-47f0-ae41-9bb9ffa6944c.png">


#### iOS
<img width="481" alt="Screenshot 2022-02-15 at 2 59 53 AM" src="https://user-images.githubusercontent.com/83179295/153957801-a6028891-e0a9-454f-b5c0-5a8515f088e5.png">



#### Android

<img width="487" alt="Screenshot 2022-02-15 at 2 58 18 AM" src="https://user-images.githubusercontent.com/83179295/153957784-81b81954-2358-4ce5-9fbf-eaf24c0a32a2.png">
